databricks/koalas/plot.py

```diff
@@ -589,7 +617,7 @@ def _get_ind(self, y):
         # 'y' is a Spark DataFrame that selects one column.
         if self.ind is None:
             min_val, max_val = y.select(
-                F.min(y.columns[0]), F.max(y.columns[0])).first()
+                F.min(y.columns[-1]), F.max(y.columns[-1])).first()
```

Comment:
I think this was existing bug in KDE @itholic since you're looking into KDE. It referred index column mistakenly before.

Reply:
Thanks for fix it. i'll pull this changes and finish my KDE work! :)